proto/beacon/p2p/v1/types.proto

```diff
@@ -212,7 +201,7 @@ message Deposit {
 
 message Exit {
   uint64 slot = 1;
-  uint32 validator_index = 2;
```

Comment:
Interesting that you were using a `uint32`. I guess another reason to move to `uint64`.

Reply:
Hah yes. I'm moving everything to `uint64` :sweat